mods/cnc/chrome/lobby-players.yaml

```diff
@@ -198,7 +198,6 @@ Container@LOBBY_PLAYER_BIN:
 							Template: ANONYMOUS_PLAYER_TOOLTIP
 						Label@NAME:
 							X: 39
-							Y: 0 - 1
```

Comment:
We'll need the same change in `TEMPLATE_EMPTY` too.

Reply:
In `TEMPLATE_NONEDITABLE_SPECTATOR` as well.